pkg/placement/membership.go

```diff
@@ -333,18 +332,46 @@ func (p *Service) disseminateOperation(ctx context.Context, target daprdStream,
 			return nil
 		}
 
-		if err := target.stream.Send(o); err != nil {
-			remoteAddr := "n/a"
-			if p, ok := peer.FromContext(target.stream.Context()); ok {
-				remoteAddr = p.Addr.String()
+		sendCtx, sendCancelFn := context.WithTimeout(ctx, 2*time.Second) // We should make this configurable
```

Comment:
This isn't a very long time, why 2 seconds?

Reply:
Yes, I wanted to discuss this value. I believe it should be configurable because it's a function of the underlying network as well as the number of dissemination messages we're expecting and how quickly can the sidecars process them; it can't be too long because it would imply we're locking all daprds for a long time (during dissemination). What would you suggest as a good default?